### DISCUSSION TITLE
Swipe to switch between notes at reading mode

### DISCUSSION THREAD

**Discussion by verifyurhuman** [damionx7/Zettel-Notes-Documentation#59] [Feature Requests]
Sometimes I read several notes at a time. When I wanna read them all, I have to navigate back to `notes list` and select another, Can you make it more simple? [This app](https://play.google.com/store/apps/details?id=com.fairapps.memorize) gives a good example. In `reading mode`, swipe left and right to switch next/previous notes, just like reading & turning pages on book. If it conflicts with current behavior, you can add a setting for user to choose. Appreciate your work.

**Comment by damionx7** [comment]
You can navigate notes during read mode by using recently implemented (v2.9.5) search feature in right drawer.
